### PR TITLE
Ejected CRA and tweaked webpack config

### PR DESCRIPTION
**Pull request by sunify**
These optimizations require ejecting CRA configs, but i think it worth it and will reduce bundle's size by ≈225 KB gzipped

### CODE REVIEW

**Comment by jondavidjohn** [comment]
Nice

I built a tool to help you analyze webpack bundles for size regressions, and report them directly to GitHub PRs. It's free for open source, so it might be worth checking out and helpful to keep these optimizations in tact now that you've ejected.

[https://packtracker.io](https://packtracker.io/?utm_campaign=links&utm_source=github&utm_medium=issueprcomment)

image

**Comment by TimDaub** [comment]
> I built a tool to help you analyze webpack bundles for size regressions, and report them directly to GitHub PRs. It's free for open source, so it might be worth checking out and helpful to keep these optimizations in tact now that you've ejected.
> 
> [https://packtracker.io](https://packtracker.io/?utm_campaign=links&utm_source=github&utm_medium=issueprcomment)

Hi @jondavidjohn, this is a great tool! We could definitely use something like this! I've created an issue to integrate a solution like: https://packtracker.io See: #199

File: package.json [hunk 1]
```diff
@@ -3,15 +3,34 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
```
Comment (TimDaub) [inline package.json]: You're adding a lot of `devDependencies` into `dependencies`. Would you mind transferring them to `devDependencies`?

File: package.json [hunk 2]
```diff
@@ -70,5 +113,55 @@
...
+      "ts",
+      "web.tsx",
+      "tsx",
+      "json",
+      "web.jsx",
+      "jsx",
+      "node"
+    ],
+    "watchPlugins": [
+      "/Users/sunify/leapdao/burner-wallet/node_modules/jest-watch-typeahead/filename.js",
```
Comment (TimDaub) [inline package.json]: How can we make sure these plugins work on other collaborators' machines?

File: package.json (hunk 1 above)
Comment (sunify) [reply package.json]: Agree (it's a default behavior of eject, forgot to move it to devDeps)

File: package.json (hunk 2 above)
Comment (sunify) [reply package.json]: LOL :D

**Comment by TimDaub** [comment]
Talked to @sunify in LeapDAO Slack about this PR. Reasons he gave for why we should move to custom script instead of CRA:

- CRA uses a good config for an average app, but burner has web3 in deps, that makes it not so average and requires some tuning
- custom configs are more easily tunable to our needs in the future

I'm pro merging this. We definitely want to reduce the burner-wallet's footprint as much as possible and this is a good first step.

@dmihal Thoughts?

**Comment by TimDaub** [comment]
@sunify would you be so kind to rebase this PR against current master such that we can merge it?
I'm proposing to approve this change.

**Comment by sunify** [comment]
@TimDaub done

**Pull request merged**